.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: false
```

Comment:
it tells travis whether or not to use the container based infrastructure.  it's not necessary for repositories enabled after 2015 (when they changed the defaults), but I am not sure when we enabled on this repository since it's pretty old, so it's safer just to always specify.